content-repo/extra-docs/articles/CoA-premium-pack-readme.md

```diff
@@ -0,0 +1,135 @@
+Read these instructions carefully to first understand the workflows that this pack executes and understand how the pack must be configured and implemented.
+ 
+# MITRE ATT&CK Tactics and Techniques
+ 
+MITRE ATT&CK is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations. 
+The framework can be used by SOC and threat intelligence analysts, threat hunters, red teamers, and defenders to better classify attacks and assess risks for their organization. 
+Organizations can use the framework to identify gaps in their defenses, prioritize them, and take the necessary actions to remediate the threat.
+ 
+Different cyber security systems classify incidents and reports based on the MITRE ATT&CK framework.
+Cortex XSOAR uses the MITRE ATT&CK feed integration to ingest the information about these techniques and sub-techniques, and many different integrations to retrieve indicators and incidents obtaining these techniques.
+ 
+# Pack Workflows
+Supported most prevalent techniques:
+1) T1071: Application Layer Protocol
+2) T1566.001: Spearphishing Attachment
+3) T1105: Ingress Tool Transfer
+4) T1059: Command and Scripting Interpreter
+5) T1059.001: PowerShell
+6) T1547: Boot or Logon Autostart Execution
+7) T1547.001: Registry Run Keys / Startup Folder"
+8) T1082: System Information Discovery"
+
+# Before You Start
+The techniques that will be handled using the playbooks in this pack are determined by the techniques ID data, which can be ingested and forwarded to the playbook by methods:
+Unit 42 and other Feed integrations - Fetch indicators from threat intelligence feeds that store MITRE ATT&CK techniques under the “Feed Related Indicators” grid field. 
+For examples - Fetching STIX reports from the Unit42 feed integration stores the techniques that are related to the threat brief under the “Feed Related Indicators” table, and can be used as a trigger to the playbooks in this pack.
+Possible indicator types to store MITRE ATT&CK techniques: STIX Report, STIX Malware, etc.
+*** This method requires a Cortex XSOAR TIM license.	
+From incident - For incident types that include techniques in the incident layout, the incident handling playbook can include the “MITRE ATT&CK - Courses of Action'' wrapper playbook as a sub-playbook. In this case, all of the associated techniques will be remediated by CoAs in addition to the specific IOCs.
+Possible incidents: Cortex XDR incidents and alerts.
+Specific technique trigger - Remediate specific techniques using their corresponding COA playbook.
+Example - techniques fetched from the MITRE ATT&CK feed.
+Job - Use a job to trigger the “MITRE ATT&CK - Courses of Action” wrapper playbook to keep the environment up to date with all relevant CoAs.
+Pack Configurations
+To configure and use the playbooks in this pack, you should do the following:
+ 
```

Comment:
Should reference the dependencies and their documentation. 

Additions for the Unit42 feed- 
*reference playbook viewer, add image as well. 
*Explain that the reports are indicators of type STIX report
*Explain that the Mitre Techniques are referenced as feed related indicators under stix report
*Each Mitre Technique is clickable, quick view will show. For more details click full details to read additional information from Mitre.  

Additions for Mitre Att&ck:
*Mention the Mitre dashboard to show the status across incidents
*Mention that without configuring this pack there will not be any data populated on the indicator (even after we create it via unit42)

Additions for PAN-OS:
*List all the license pre-requisites and explain they are needed for certain actions. 
*Highlight the approval flow for policy changes
*Explain FW vs Panorama

@mayagoldb @kirbles19 @yaron-libman

Reply:
Unit 42 part should also be emphasized in the Unit 42 Pack documentation and link to this pack.